core/src/main/java/org/apache/iceberg/hadoop/HadoopTables.java

```diff
@@ -203,13 +204,31 @@ public boolean dropTable(String location, boolean purge) {
     }
   }
 
+  private Map<String, String> encryptionConfigFromConf() {
```

Comment:
I don't think this should update `HadoopTables`. New features should not be added to that interface.

Reply:
@rdblue , not sure I understand this one, need your help to clarify: do you mean for `HadoopTables` we don't need to incorporate `EncryptionManagerFactory` initialisation logic or we shouldn't load the catalog properties from Hadoop conf?